pkg/tests/api/alerting/api_ruler_test.go

```diff
@@ -428,7 +428,7 @@ func TestAlertRuleConflictingTitle(t *testing.T) {
 		require.NoError(t, err)
 
 		assert.Equal(t, http.StatusInternalServerError, resp.StatusCode)
-		require.JSONEq(t, `{"message": "failed to update rule group: failed to add or update rules: a conflicting alert rule is found: rule title under the same organisation and folder should be unique"}`, string(b))
+		require.JSONEq(t, `{"message": "failed to update rule group: failed to add rules: a conflicting alert rule is found: rule title under the same organisation and folder should be unique"}`, string(b))
```

Comment:
Does this mean we don't have an integration test for update? Can we add one?

Reply:
We have one, but it's testing a successful request. We don't really have something that could fail logic-wise like a colliding rule title on an update that we have not already tested somewhere else.

Reply:
What we could test is having two rules with different names and trying to rename one like the other, this should then fail with a similar error. 

I'll add this.